### PR TITLE
Add docs examples for invalid-* and isinstance-*

### PR DESCRIPTION
**Pull request by clavedeluna**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |

| ✓   | :scroll: Docs          |

## Description

Refs #5953

### CODE REVIEW

File: doc/data/messages/i/invalid-metaclass/bad.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class InvalidApple(metaclass=int):  # [invalid-metaclass]
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/i/invalid-metaclass/bad.py:1]:
```suggestion
class Apple(metaclass=int):  # [invalid-metaclass]
```

File: doc/data/messages/i/invalid-metaclass/good.py [hunk 2]
```diff
@@ -1 +1,2 @@
-# This is a placeholder for correct code for this message.
+class ValidApple(type):
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/i/invalid-metaclass/good.py:1]:
```suggestion
class Plant:
    pass

class Apple(Plant):
```

File: doc/data/messages/i/invalid-repr-returned/bad.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class BadRepr:
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/i/invalid-repr-returned/bad.py:1]:
```suggestion
class Repr:
```

File: doc/data/messages/i/invalid-repr-returned/good.py [hunk 4]
```diff
@@ -1 +1,5 @@
-# This is a placeholder for correct code for this message.
+class GoodRepr:
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/i/invalid-repr-returned/good.py:1]:
```suggestion
class Repr:
```

File: doc/data/messages/i/invalid-str-returned/bad.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+class BadStr:
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/i/invalid-str-returned/bad.py:1]:
```suggestion
class Str:
```

File: doc/data/messages/i/invalid-str-returned/good.py [hunk 6]
```diff
@@ -1 +1,5 @@
-# This is a placeholder for correct code for this message.
+class GoodStr:
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/i/invalid-str-returned/good.py:1]:
```suggestion
class Str:
```

**Review by Pierre-Sassoulas — CHANGES_REQUESTED** [review]
Nitpick : if the differences between good/bad are only problematic code construct vs good code construct it's easier to understand the example when playing "spot the difference".

File: doc/data/messages/i/invalid-metaclass/good.py (hunk 2 above)
Comment (Pierre-Sassoulas) [reply doc/data/messages/i/invalid-metaclass/good.py:1]: Do you agree with this suggestion @clavedeluna ?

Comment (clavedeluna) [reply doc/data/messages/i/invalid-metaclass/good.py:1]: Yep! My only thought when seeing this is that if one of the goals of this docs project is to outsource to contributors adding these examples, then maybe the variety that contributors bring to the table is worth keeping. I don't mind either way.

**Review by Pierre-Sassoulas — APPROVED** [review]

**Review by Pierre-Sassoulas — APPROVED** [review]

**Pull request merged**